samples/nrf9160/custom_trace_medium/modem_trace_medium_custom/modem_trace_medium_custom.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <zephyr/kernel.h>
+#include <debug/cpu_load.h>
+#include <modem/trace_medium.h>
+#include <sys/errno.h>
+
+#define PRINT_PERIOD_MSEC CONFIG_CUSTOM_MEDIUM_PRINT_PERIOD_MSEC
+
+static uint32_t num_bytes;
+static uint32_t num_bytes_prev;
+static bool show_cpu_load;
+
+static void print_stats(struct k_work *item);
+
+K_WORK_DELAYABLE_DEFINE(print_stats_work, print_stats);
+
+static void print_stats(struct k_work *item)
+{
+	float throughput;
+	float cpu_load_percent;
+
+	ARG_UNUSED(item);
+
+	throughput = (num_bytes - num_bytes_prev)/(PRINT_PERIOD_MSEC/1000.0);
```

Comment:
Maybe this is unnecessary overcomplicating the sample. However, do we want a more precise elapsed time here, using e.g. k_uptime_get. Although the sample is quite simple, the time is not necessarily precisely `PRINT_PERIOD_MSEC`.
```suggestion
	throughput = (num_bytes - num_bytes_prev)/(k_uptime_get_32()/1000.0);
```